modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -173,20 +169,29 @@ async function invokeServer<T>(
 					}),
 					method: 'POST',
 				})
-					.then((response) => response.text())
-					.then((text) => {
-						const json = JSON.parse(text);
-
-						if (json.error) {
-							resolve({
-								error: json.error,
-							});
-						}
-						else {
-							resolve({
-								result: json,
-							});
+					.then(async (response) => {
+						if (!response?.ok) {
+							if (response?.status === 404) {
+								reject(
+									`Feature Flag "${Liferay.Util.escapeHTML(
```

Comment:
Do we really need to escape this? This key is just passed in code from tests. It's never coming from a user, right?

Reply:
nice catch. I'll remove it